airflow/www/app.py

```diff
@@ -88,6 +88,8 @@ def create_app(config=None):
             models.Connection, Session, name="Connections", category="Admin"))
         av(vs.VariableView(
             models.Variable, Session, name="Variables", category="Admin"))
+        av(vs.XComView(
```

Comment:
Please make sure this is not viewable from a normal user's perspective as this can expose sensitive information